decidim-admin/spec/system/admin_manages_newsletters_spec.rb

```diff
@@ -113,6 +113,24 @@
       visit decidim_admin.preview_newsletter_path(newsletter)
       expect(page).to have_content("Hello Sarah Kerrigan! Relevant content.")
     end
+
+    it "sends a test email to the logged user" do
```

Comment:
The titles of the specs are a bit weird (specially the `it "admin clicks on this index button and receives the test newsletter"` in line 126).

I think that using contexts for clarifying can improve the legibility a bit: 

```suggestion
    context "when admin clicks on the 'send me a test email' button" do
      it "sends a test email" do
```

Reply:
Done

Reply:
I think it wasn't applied correctly, can you check it out again please?

Reply:
Sorry. Now it's done.